src/utils/shim.js

```diff
@@ -21,6 +41,7 @@ if (typeof localStorage !== 'undefined') {
   localStorage.debug = isDev ? '*' : ''
 }
 
+export const isProdEnv = NETWORK_PROVIDER === 'homestead'
```

Comment:
shim.js file is auto-generated, once you install any npm package this function will be deleted

Reply:
😢 any other suggestion for this then? @jsidorenko

Reply:
ok moved to `utils/environment` for now

Reply:
just put in any other file from utils folder